module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -1744,6 +1746,18 @@ public function getConfig($function, $params = [])
                 ],
                 'default_sort' => '+due_date',
             ];
+        } elseif ('Holdings' === $function) {
+            $limitByType = $this->config['Holdings']['itemLimitByType'];
```

Comment:
This needs null coalescing (like ` ?? []`), otherwise this will cause an `Undefined array key` error if the setting isn't found.

Reply:
Thanks @EreMaijala this is included in my followup